### PR TITLE
Fix selectors for MM 10.x and update default version to 10.15.0

### PR DESCRIPTION
**Pull request by lift0xff**
this PR is now ready for merging. 9.7.1 is no longer available from the download URL.

### CODE REVIEW

**Comment by pcardosolei** [comment]
cool stuff. thanks. will launch a version this.

File: pages/metamask/first-time-flow-page.js [hunk 1]
```diff
@@ -23,14 +23,14 @@ module.exports.metametricsPageElements = {
...
+const secretWordsInput = `${firstTimeFlowFormPage} .import-srp__srp input[type="password"]`;
 const passwordInput = `${firstTimeFlowFormPage} #password`;
 const confirmPasswordInput = `${firstTimeFlowFormPage} #confirm-password`;
-const termsCheckbox = `${firstTimeFlowFormPage} .first-time-flow__terms`;
-const importButton = `${firstTimeFlowFormPage} .first-time-flow__button`;
+const termsCheckbox = `${firstTimeFlowFormPage} #create-new-vault__terms-checkbox`;
+const importButton = `.create-new-vault__submit-button`;
 const newPasswordInput = `${firstTimeFlowFormPage} #create-password`;
-const newSignupCheckbox = `${firstTimeFlowFormPage} .first-time-flow__checkbox`;
+const newSignupCheckbox = `${firstTimeFlowFormPage}  .first-time-flow__checkbox`;
```
Comment (al1y) [inline pages/metamask/first-time-flow-page.js:33]: Just found that these selector changes work for the import wallet but break the create wallet flow

Comment (pcardosolei) [reply pages/metamask/first-time-flow-page.js:33]: hey @al1y 

I'm kinda overwhelmed at the moment.
If you can add the solution I would be be grateful.

thanks.
Paulo

Comment (al1y) [reply pages/metamask/first-time-flow-page.js:33]: Same, just wanted to note while here, I have a patch that I'm working with locally to keep track of changes, will round back here when I get a chance

Comment (pcardosolei) [reply pages/metamask/first-time-flow-page.js:33]: Thanks @al1y 

much appreciated. 

P.S - I need to think on a way to have some sort of CI/CD for this plugin.

**Pull request merged**